uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -483,6 +499,20 @@ define([
       };
 
       /**
+       * Checks if given status is applicable to manager
```

Comment:
not "applicable", doesn't make sense in this sense. you are checking if the status is available for the manager to use